pkg/cloud/aws/provider.go

```diff
@@ -1615,9 +1615,10 @@ func (aws *AWS) getAllAddresses() ([]*ec2Types.Address, error) {
 
 	// Get volumes from each AWS region
 	for _, r := range regions {
+		region := r // make a copy of r to avoid capturing loop variable
```

Comment:
I prefer doing `r := r` to avoid having two vars that may or may not point to the same thing, but that doesn't block this change. Good find

Reply:
Go loop captures strike again! It stinks they're about to rollout a fix for this gotcha in 1.22 I think.